src/api/auth.ts

```diff
@@ -0,0 +1,67 @@
+import axios from 'axios';
+import {SERVER_URL, USER_DEFAULT_IMG} from 'constant/constant';
+
+export const headers = {
+  'content-type': 'application/json',
+  serverId: process.env.REACT_APP_SERVER_ID,
+};
+
+export const authHeaders = () => {
+  const accessToken = localStorage.getItem('accessToken');
+  return {
+    'content-type': 'application/json',
+    serverId: process.env.REACT_APP_SERVER_ID!,
+    Authorization: `Bearer ${accessToken}`,
+  };
+}; // 토큰이 필요한 api에는 해당 헤더를 사용하시면 됩니다!
+
+export const postSignIn = async (id: string, pw: string) => {
+  try {
+    const response = await axios.post(
+      `${SERVER_URL}/login`,
+      {id: id, password: pw},
```

Comment:
이렇게 동일한 파라미터명-변수 일때는
```suggestion
      {id, password: pw},
```

이렇게 줄여써도 됩니다! 웹스톰이나 이런 에디터에서는 아예 이렇게 하라고 워닝 주기도 하더라고요.